database/buildSchema/36_notification.sql

```diff
@@ -8,7 +8,9 @@ CREATE TABLE public.notification (
     subject varchar,
     message varchar,
     attachments varchar[],
+    timestamp timestamptz DEFAULT CURRENT_TIMESTAMP NOT NULL,
```

Comment:
Would this field get updated when the email is sent (on Update that runs to set 2 variables `email_sent` & `email_server_log`, from my understanding it would be only set on Insert

Reply:
I think the "timestamp" is the timestamp of the notification record, not the email.

I'm not sure that the timestamp of the email is super-important, it's just the result that matters, isn't it?

(I guess we could append an additional timestamp to the email_server_log string if it seems important?)